Gaijinizer patch/Map449.txt

```diff
@@ -4,111 +4,146 @@
 
 \\N[1]くんの世界と、私の世界は違うんだね。
 とてもよく似ているけど、部分的に…。
-> CONTEXT: Map449/events/1/pages/0/5/Dialogue/enemy15 < UNTRANSLATED
-
+> CONTEXT: Map449/events/1/pages/0/5/Dialogue/enemy15
+Oh, that's right.
+Your world is different from mine, isn't it?
+While they're quite similar, there are parts that
+aren't the same.#Ron
 > END STRING
 
 > BEGIN STRING
 『悪魔弾』とか、『Ｍ.Ｏ.Ｗ』とか…
 
 使う単語は同じなのに、その立場や意味が違ってるだなんて
 とっても不思議…
-> CONTEXT: Map449/events/1/pages/0/10/Dialogue/enemy15 < UNTRANSLATED
+> CONTEXT: Map449/events/1/pages/0/10/Dialogue/enemy15
+Like "Demon Rounds", and "M.O.W."...
 
+We have the same words but their meanings and
+uses are completely different.#RonIffy
 > END STRING
 
 > BEGIN STRING
 ああ……
 
 まさか\\C[2]『人間と悪魔が戦争のあと共生している世界』\\C[0]なんて
 戦争真っ只中のこっちの世界じゃ、想像もできないな…
-> CONTEXT: Map449/events/1/pages/0/15/Dialogue/kazuya < UNTRANSLATED
-
+> CONTEXT: Map449/events/1/pages/0/15/Dialogue/kazuya
+Yeah... For someone like me whose world is in the
+middle of the war, it's hard to imagine \\C[2]a world
+in which demons and humans co-exist together.#Ron
 > END STRING
```

Comment:
Possibly "the war" -> "war" (unless they're implying something like "The Great War" or something to that effect)